cmd/mapper/main.go

```diff
@@ -99,6 +99,31 @@ func contains(slice []string, item string) bool {
 	return false
 }
 
+func printHelp() {
```

Comment:
What do you think of rewriting these 2 functions using multi-line string?  
I guess it would improve the readability and make it easier to update in the future.

```
func printHelp() {
	helpText := `Kubernetes MicroLens - A lightweight Kubernetes resource visualization tool

Usage:
  k8s-microlens [flags]

Flags:
  -n, --namespace string     Process only the specified namespace
  --exclude-ns string        Exclude specified namespaces (can be specified multiple times)
  -h, --help                 Show help message
  -v, --version              Show version information

Examples:
  # Show resources in all namespaces
  k8s-microlens

  # Show resources in specific namespace
  k8s-microlens -n default

  # Exclude specific namespaces
  k8s-microlens --exclude-ns kube-system --exclude-ns kube-public
`
	fmt.Println(helpText)
}

func printVersion() {
	versionText := `Kubernetes MicroLens v0.1.0
A lightweight Kubernetes resource visualization tool

Author: Marcus Bergo <marcus.bergo@gmail.com>
Repository: https://github.com/mbergo/k8s-microlens
`
	fmt.Println(versionText)
}
```